indra/llappearanceutility/llbakingavatar.h

```diff
@@ -114,9 +112,8 @@ class LLBakingAvatar : public LLAvatarAppearance
  **                    STATE
  **/
 public:
-    /*virtual*/ bool    isSelf() const { return true; }
-    /*virtual*/ bool    isValid() const { return TRUE; }
-    /*virtual*/ bool    isUsingBakedTextures() const { return TRUE; }
```

Comment:
another vanishing method

Reply:
Those had vanished from the base long ago, effectively doing nothing (and generating override errors: "This method does not override something") 
From my understanding was this all "must have this to inherit, so dummy override this" methods.

You are right I should have included the reasoning in the fimmut comment. I will update accordingly